ee/clickhouse/queries/experiments/secondary_experiment_result.py

```diff
@@ -62,7 +62,7 @@ def get_results(self):
         else:
             raise ValidationError("Secondary metrics need to be funnel or trend insights")
 
-        return {"result": variants}
+        return variants
```

Comment:
@neilkakkar I changed the structure here because I wanted to use this to send secondary metric results back in the `experiment/results` call as one instead of making X many calls and having them load in different states on the frontend. Let me know if this makes sense and then I'll fix the backend tests accordingly for this

Reply:
Hmm, the reason they're not in one call is because this can get pretty slow: Not only are you querying for a trend/funnel breakdown, but also the significance calculation.

Add to that a few more result calls, and you become 3-5x slower. I think it's fine to break this down, and have small spinners for each of the secondary metrics, that load later than the main big one.

Thoughts?

Reply:
Also, I'd guess secondary metrics only come into play when results are significant and you're making a final decision. So, main metric should always have the priority + major screen time.


(Will take a look at this PR in a bit!)

Reply:
Alrighty I'll redo the way the frontend loads the results